tools/CMakeLists.txt

```diff
@@ -164,32 +165,45 @@ if(JPEGXL_ENABLE_TOOLS)
     gflags
   )
 
-add_executable(cjpeg_hdr
-  cjpeg_hdr.cc
-)
-target_link_libraries(cjpeg_hdr
+  add_executable(cjpeg_hdr
+    cjpeg_hdr.cc
+  )
+  target_link_libraries(cjpeg_hdr
     box
     jxl-static
     jxl_extras-static
     jxl_threads-static
-)
+  )
 
   # Main decompressor.
   add_library(djxltool STATIC
     djxl.cc
     speed_stats.cc
   )
   target_link_libraries(djxltool
-      box
-      jxl-static
-      jxl_extras-static
-      jxl_threads-static
+    box
+    jxl-static
```

Comment:
Uh, that doesn't seem right.

Reply:
I didn't change anything except the indentation...